### PR TITLE
Release 1.10.0

### PR DESCRIPTION
**Pull request by MichaelAquilina**
@rnc

### CODE REVIEW

**Comment by rnc** [comment]
@MichaelAquilina  I still have #86 to finish

**Comment by MichaelAquilina** [comment]
That's ok - I won't merge this in before the other one is done :)

**Comment by rnc** [comment]
@MichaelAquilina Do you know why the tests are failing? With #91 merged in, the tests pass locally for me.

**Comment by MichaelAquilina** [comment]
@rnc yep dw about it. It's because I havent tagged the release yet. The linter step is failing because it's checking that the `git tag` == `version in the plugin` == `version in the changelog`.

Once we get your last change in I will tag the commit and the build will start passing again :)

**Comment by rnc** [comment]
@MichaelAquilina Ok thanks ; last change is ready for merge I think in #91

**Comment by MichaelAquilina** [comment]
@rnc give this PR a final look and if you are happy with the changes I'll go ahead and merge it into master

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,6 +1,11 @@
 Changelog
 =========
 
+1.10.0
+------
+* Scan for requirements files recursively when setting up a virtualenv (#88 - fix by @nrc)
+* Fix bug when `rm` was aliased to a different behaviour (#87 - fix by @rnc)
+
```
Comment (rnc) [inline CHANGELOG.rst]: Add details on #86 e.g.
```suggestion
*  Invoke pip install in editable (i.e. development) mode.
```

**Comment by rnc** [comment]
@MichaelAquilina  One comment otherwise LGTM :-)

**Comment by MichaelAquilina** [comment]
Thanks for all the changes @rnc :tada:

**Pull request merged**